### DISCUSSION TITLE
Problem with iMac 4K and resolution

### DISCUSSION THREAD

**Discussion by zabdoul60460** [airsdk/Adobe-Runtime-Support#2316] [Q&A] [answered]
Hi,
Why on iMac 4K my app only opens in low resolution ? I coud not uncheck the checkbox on app properties.
May I update the code ?
I works with Harman Adobe Air 33.1.1.821. The code is compiling with VSCode and AS3Mxml.
Thanks for helping.
Best regards.

**Comment by ajwfrost** [comment, marked as answer]
Hi
Are you setting the `requestedDisplayResolution` value in the app descriptor file?
https://airsdk.dev/docs/building/application-descriptor-files/elements/initialWindow#requesteddisplayresolution
thanks

**Reply by zabdoul60460** [reply]
This option is not in my app descriptor file. I'll add it and give you some news.
Best regards

**Reply by zabdoul60460** [reply]
Thanks for your help.
With this setting it works !!!
Best regards
